sdkexamples/go.mod

```diff
@@ -0,0 +1,144 @@
+module github.com/helm/helm-www/sdkexamples
```

Comment:
I put the source in this 'sdkexamples' directory. As it is meant to be a buildable go project. Suggestions on how to better structure this welcome